distributions/otelcol-minimal/manifest.yaml

```diff
@@ -0,0 +1,18 @@
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/minimal
+  name: otelcol-minimal
+  description: OpenTelemetry Collector Minimal
+  version: 0.109.0
+  output_path: ./_build
+  otelcol_version: 0.109.0
+
+exporters:
```

Comment:
I had a similar distribution myself and it was very similar to this one at first, but found that I was missing a couple of things:

- at least one auth extension
- pprof extension
- debug exporter

Without those, I think the Collector cannot be properly monitored, secured, and configured (debug exporter is useful when setting up pipelines in general).

I also ended up adding this, but I think we should have v1 without it, adding it just after we have the healthcheck v2 in place:

- healthcheckextension

Reply:
We discussed this on the 2024-09-18 Collector stability meeting; we can start with just otlpreceiver, otlpexporter and otlphttpexporter for now and open dedicated issues to add the other components mentioned here at some point (which may be after 1.0 and instead in a later 1.x release). Once we merge this PR I will file issues to add these components to the distro.